protocols/direct-fund/direct-fund_test.go

```diff
@@ -149,11 +176,11 @@ func TestCrank(t *testing.T) {
 	}
 
 	// Manually progress the extended state by collecting prefund signatures
-	o.(DirectFundObjective).preFundSigned[0] = true
-	o.(DirectFundObjective).preFundSigned[1] = true
+	o.(DirectFundObjective).C.AddSignedState(o.(DirectFundObjective).C.PreFundState(), correctSignatureByAliceOnPreFund)
+	o.(DirectFundObjective).C.AddSignedState(o.(DirectFundObjective).C.PreFundState(), correctSignatureByBobOnPreFund)
```

Comment:
This is gnarly, and I have a neat way to tidy it, but it doesn't fit nicely as a suggestion (touches a bunch of lines). Will punt to a small follow-up PR.